protoc-gen-openapiv3/internal/genopenapiv3/convertors.go

```diff
@@ -0,0 +1,97 @@
+package genopenapiv3
+
+import (
+	"github.com/getkin/kin-openapi/openapi3"
```

Comment:
I'd prefer not to add a third party dependency like this, could we define these types ourselves?